15_transfer_learning.py

```diff
@@ -167,7 +167,9 @@ def train_model(model, criterion, optimizer, scheduler, num_epochs=25):
 
 # Parameters of newly constructed modules have requires_grad=True by default
 num_ftrs = model_conv.fc.in_features
-model_conv.fc = nn.Linear(num_ftrs, 2)
+
+# Creating a FC layer of the same size as our categories

```

Comment:
Please move this to line 168 and use this comment:
# Creating a FC layer of the same size as our categories.
# Parameters of newly constructed layers have requires_grad=True by default

Reply:
Looks like a # is rendered as headline here in markdown :D